test/unparse/exp/lit.local.cfg.py

```diff
@@ -0,0 +1,4 @@
+config.excludes = [
```

Comment:
Should we move these into the parent directory somehow since it's the same across each directory?